client/src/pages/Homepage/TopSection.jsx

```diff
@@ -132,12 +132,12 @@ const TopSection = () => {
 			<Cover>
                 <H3>{t.homepageMainTopTitle}</H3>
 				<H1>{t.homepageMainTitle}</H1>
-				<SearchBox />
+				<SearchBox backgroundColor="#4D20B2" wrapperPadding="20px" />
```

Comment:
You can add the stylying to the SearchBox component itself

Reply:
@GalGend we need to use props because this component is used in 3 different places and they have different background color and padding